patch/ps_command.go

```diff
@@ -0,0 +1,176 @@
+package patch
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"regexp"
+	"strings"
+	"unicode/utf8"
+
+	"golang.org/x/text/encoding/simplifiedchinese"
+	"golang.org/x/text/transform"
+)
+
+func decodePowerShellOutput(input []byte) (string, error) {
+	if utf8Output := string(input); strings.HasPrefix(utf8Output, "\uFEFF") {
```

Comment:
Can we always use English? Or do we have to use different code to support different languages?

This code is fragile. Notice that Windows can have dozens of UI languages